lighthouse-core/gather/gatherers/styles.js

```diff
@@ -98,8 +98,9 @@ class Styles extends Gatherer {
         }).then(content => {
           const styleHeader = this._activeStyleHeaders[sheetId];
           styleHeader.content = content.text;
-          styleHeader.parsedContent = getCSSPropsInStyleSheet(
-              parser.parse(styleHeader.content));
+          const parsedContent = parser.parse(styleHeader.content);
+          styleHeader.parsedContent = parsedContent.error ?
```

Comment:
Since this is the stylesheet gatherer, we should at least log that we got a stylesheet we couldn't parse. I'd say add a `log.warn('Styles Gatherer', '....${parsedContent.error}'`?